test/ui_test_label_studio.py

```diff
@@ -0,0 +1,190 @@
+        # iframe_ = self.driver.find_element_by_xpath(r'//iframe[@srcdoc, "<html><head><!--EditorCSS--><linkhref=&quot;static/editor/css/main.a1b00718.css&quot;rel=&quot;stylesheet&quot;><!--EditorJS--><scriptsrc=&quot;static/editor/js/main.12760629.js&quot;></script><style>body::-webkit-scrollbar-track{-webkit-box-shadow:inset002pxrgba(178,178,178,0.3)!important;background-color:#F5F5F5!important;}body::-webkit-scrollbar{width:2px!important;height:2px!important;background-color:#F5F5F5!important;}body::-webkit-scrollbar-thumb{background-color:#c3c3c3!important;border:2pxsolid#b4b4b4!important;}#label-studiodiv[class^=&quot;App_editor&quot;],div[class*=&quot;App_editor&quot;]{width:98%!important;min-width:98%!important;max-width:98%!important;}#label-studio.ls-menu{width:100%;}#label-studio.ls-segment{top:0;}</style></head><body><scriptsrc=&quot;static/js/jquery.min.js&quot;></script><scriptsrc=&quot;static/js/lsf-sdk.js&quot;></script><!--Editor--><divclass=&quot;uicontent&quot;><divid=&quot;label-studio&quot;style=&quot;width:100%;&quot;><divclass=&quot;uiloaderactiveinlinecentered&quot;></div></div></div><script>window.onload=function(){vartask={&quot;id&quot;:42,&quot;data&quot;:{&quot;image&quot;:&quot;static\/samples\/sample.jpg&quot;},&quot;completions&quot;:null,&quot;predictions&quot;:null,&quot;project&quot;:1,&quot;created_at&quot;:&quot;2019-02-06T14:06:42.000420Z&quot;,&quot;updated_at&quot;:&quot;2019-02-06T14:06:42.000420Z&quot;};varLS=newLabelStudio(&quot;label-studio&quot;,{config:`<View><Headervalue=&quot;Selectlabelandstarttoclickonimage&quot;></Header><Imagename=&quot;image&quot;value=&quot;$image&quot;></Image><PolygonLabelsname=&quot;label&quot;toName=&quot;image&quot;strokeWidth=&quot;3&quot;pointSize=&quot;small&quot;opacity=&quot;0.9&quot;><Labelvalue=&quot;Airplane&quot;background=&quot;red&quot;></Label><Labelvalue=&quot;Car&quot;background=&quot;blue&quot;></Label></PolygonLabels></View>`,user:{pk:1,firstName:&quot;Awesome&quot;,lastName:&quot;User&quot;},task:_convertTask(task),interfaces:[&quot;basic&quot;,&quot;side-column&quot;,&quot;predictions&quot;],onLabelStudioLoad:function(LS){varc=LS.completionStore.addCompletion({userGenerate:true});LS.completionStore.selectCompletion(c.id);}});};</script></body></html>"]')
+        # self.driver.switch_to.frame(iframe_)
+        # self.driver.find_element_by_xpath(self.setup_label_car)
+        # time.sleep(5)
+        #
+        # Click on Label Airplane
+        label_car_btn = self.wait.until(EC.element_to_be_clickable((By.CLASS_NAME, self.setup_label_car)))
```

Comment:
so does it work with iframe now?
for Setup I think it's enough to just select some predefined template and click Save

Reply:
About Setup page: thanks for suggestion) Now script clicks on Save button and saves default template.
After discussions I can add your wanted predefined template)